### PR TITLE
page working with all the functions

### PR DESCRIPTION
**Pull request by Svartstein**
## In this pull request:
- Added functions to check and set the tasks status.
- Added function to clean all the completed tasks.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -22,7 +24,7 @@ const printList = () => {
         tasks += `
           <li id = "${task.index}" class="item">
             <label>
-              <input type="checkbox" name="chk${task.index}" id="chk${task.index}">
+              <input class="checked" type="checkbox" name="chk${task.index}" id="chk${task.index}">
               <input class="edit borderless fit hidden" type="text" name="edit${task.index}" id="edit${task.index}" placeholder="Edit task...">
               <p id="task${task.index}">${task.description}</p>
             </label>
```
Comment (mirouhml) [inline src/index.js]: - [x] Please know that there is an issue with this function:
  image
  I wasn't able to test anything, so kindly fix this issue so we can test the functionalities of your project.

File: src/index.js [hunk 2]
```diff
@@ -84,17 +111,22 @@ const printList = () => {
...
   toDoList = getToDoList();
   const newIndex = toDoList.length + 1;
   const task = new Task(newTaskInput.value, newIndex);
   task.Add();
   newTaskInput.value = '';
   printList();
+  // checkCompleted();
 };
 
 // Add new Task
```
Comment (mirouhml) [inline src/index.js]: - [x] Please know that adding a new task isn't working at all kindly fix the issue as I wasn't able to test anything.

File: src/index.js [hunk 3]
```diff
@@ -41,41 +43,66 @@ const printList = () => {
...
 
-      // console.log(document.querySelector(`#task${taskIndex}`));
       printList();
     });
   });
+
+  // delete selected task
   document.querySelectorAll('.delete-task').forEach((dtb) => {
     dtb.addEventListener('click', (e) => {
       const task = new Task();
```
Comment (mirouhml) [inline src/index.js]: [OPTIONAL]
- [x] Deleting also isn't working for some reason, here is the error:
  image

File: src/index.js [hunk 4]
```diff
@@ -41,41 +43,66 @@ const printList = () => {
...
       const taskElement = document.getElementById(`${taskIndex}`);
-      // console.log(taskElement);
-
-      document.querySelector(`#edit${taskIndex}`).classList.remove('hidden');
+      const editTask = document.querySelector(`#edit${taskIndex}`);
+      editTask.classList.remove('hidden');
       document.querySelector(`#task${taskIndex}`).classList.add('hidden');
 
       taskElement.querySelector('.edit-manager').classList.remove('hidden');
       taskElement.querySelector('.list-editor').classList.add('hidden');
```
Comment (mirouhml) [inline src/index.js]: [OPTIONAL]
- [x] Editing also isn't working I am getting the same error as the delete error:
  image

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @Swarzstein, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — CHANGES_REQUESTED** [review]
### Hi @Swarzstein, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Ntohing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**